components/Anchor.js

```diff
@@ -13,7 +13,7 @@ const InvisibleAnchor = styled.div.attrs({
   visibility: hidden;
   height: 0;
 
-  top: ${rem(-20)};
+  top: ${rem(-80)};
 
   ${mobile(css`
     top: ${rem(-90)};
```

Comment:
Have you checked whether this needs some changes on mobile?

Reply:
Oh, No. I'll check.

Reply:
It's now good on both mobile and desktop IMO.